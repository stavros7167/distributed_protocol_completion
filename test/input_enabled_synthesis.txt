environment E {
  inputs []
  outputs [a, b]
  initial e0
  e0 a! e0
  e0 b! e0
}

process P {
  states [p0]
  inputs [a, b]
  input_enabled [a, b]
  outputs []
  initial p0
}
