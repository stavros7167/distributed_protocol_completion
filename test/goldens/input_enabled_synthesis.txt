Parsing file test/input_enabled_synthesis.txt
# automata read: 2
SOLUTION #1 FOUND:
** Solution 1 **
('P', 'p0', 'a?', 'p0')
('P', 'p0', 'b?', 'p0')
{'candidate transitions': 2, 'iterations': 1}
Search finished.
